test/kokoro/xds.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -exu -o pipefail
+if [[ -f /VERSION ]]; then
+  cat /VERSION
+fi
+
+cd github
```

Comment:
Where is this to be executed from?

Maybe document that at the top.  Or just a quick note that only CI is supposed to run this file.  Or a `README.md` in this directory.

Reply:
Added README.md. Kokoro will clone the repo into the github/ directory and then run from its parent via <magic>.